### ISSUE TITLE
Vertebral zones specific field correlation for more effective unwrap

### ISSUE THREAD

**Issue by Laura2305** [NordicMRspine/MRINavigator.jl#55]
I think we should add an optional input parameter defining the vertebral levels covered by the acquisition. We can use this to obtain the correlation between the respiratory trace and the field variations, instead of measuring it, as the measurement is often inaccurate.
This only concerns the phase unwrap function.

**Comment by alexjaffray** [comment]
That is a good idea, both methods can probably coexist. Perhaps we should lay out the logic so that there is a possibility of choosing the most reliable method if both are provided.

**Comment by Laura2305** [comment]
Yes, I agree. I have an idea of how to do it. I opened the issue as a reminder for when I have time :)
